NavigationReactNative/src/ios/NVSceneController.m

```diff
@@ -49,6 +49,16 @@ - (void)viewWillAppear:(BOOL)animated
 
     [navigationBar updateColors];
 
+    NSInteger crumb = [self.navigationController.viewControllers indexOfObject:self];
+    UIViewController *previousViewController = crumb > 0 ? [self.navigationController.viewControllers objectAtIndex:crumb - 1] : nil;
+    if (previousViewController != nil) {
+        if (navigationBar.backTitle == nil) {
+            previousViewController.navigationItem.backBarButtonItem = nil;
+        } else {
+            previousViewController.navigationItem.backBarButtonItem = [[UIBarButtonItem alloc] initWithTitle:navigationBar.backTitle style:UIBarButtonItemStyleDone target:nil action:nil];
```

Comment:
Think we should use `UIBarButtonItemStylePlain` instead of `UIBarButtonItemStyleDone`. Same applies to `NVNavigationBarView`